### ISSUE TITLE
Not working with Umbraco 7.7.2

### ISSUE THREAD

**Issue by msinnott22** [warrenbuckley/Analytics#53]
I've recently upgrade one of my websites from 7.5 to the latest version (7.7.2) and the Analytics Package no longer works. 

I've attached a screenshot depicting what I am seeing. 

![image](https://user-images.githubusercontent.com/2410904/31307045-22ebadec-ab54-11e7-9dd2-fc5bf65fce47.png)

Thanks,
Mike

**Comment by warrenbuckley** [comment]
All these labels are for translation and the language your backoffice user is set to does not have the translations set for that language @msinnott22

**Comment by msinnott22** [comment]
Hi @warrenbuckley, 

Thanks for your prompt response. 

My user is set to English (United Kingdom), I'm guessing this isn't mapping to the en.xml Lang file? 

![image](https://user-images.githubusercontent.com/2410904/31333636-bfe52d1e-ace2-11e7-9c24-ff0647805f00.png)

Cheers,
Mike

**Comment by warrenbuckley** [comment]
Odd, tried in English US to see if it comes back?

**Comment by msinnott22** [comment]
Yeah, tried in both and still get the same issue. I've checked and I do have the lang files on my server to.

Anything else I can try or logs to look into?

**Comment by warrenbuckley** [comment]
Check you have files in `App_Plugins/Analytics/Config/lang` like so
https://github.com/warrenbuckley/Analytics/tree/develop/Analytics/App_Plugins/Analytics/config/lang

**Comment by msinnott22** [comment]
Yeah, I don't have all of them but I do have en & en_us:

![image](https://user-images.githubusercontent.com/2410904/31336370-7ba7b412-acee-11e7-85e0-d45f04201564.png)

**Comment by warrenbuckley** [comment]
Odd!
Do they contain anything as it should be picking up values here.

Also have you changed the Client Dependency Framework version in Umbraco & ensure your browser is not caching anything?

**Comment by msinnott22** [comment]
Yeah, the files themselves look fine. 

I've not changed the Client Dependency Framework but I will look into this. I've tried various browser and cleared everything, I get the same issue. 

I'll comment again once I have tested.

**Comment by msinnott22** [comment]
Hi @warrenbuckley, 
I tested the above and I'm still getting the issue. Would it be due to the lang files changing to en_gb.xml from en.xml? 
Cheers,
Mike

**Comment by purplepiranha** [comment]
I get the same issue running v7.7.7 and also the analytics icon does not appear in the menu. You may be interested to see the script error that I get:
![image](https://user-images.githubusercontent.com/15064668/34320932-544105b6-e7fb-11e7-8a25-fab31fd4b908.png)

**Comment by robgha01** [comment]
This is the same issue as https://github.com/warrenbuckley/Analytics/issues/28
when i reload it sometimes work and sometimes throws that error.
